src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -213,7 +213,14 @@ public static string CleanTitle(string title)
 
         public static string TitleThe(string title)
         {
-            return TitlePrefixRegex.Replace(title, "$2, $1$3");
+            if (string.IsNullOrWhiteSpace(title))
+            {
+                return string.Empty;
+            }
+            else
+            {
+                return TitlePrefixRegex.Replace(title, "$2, $1$3");
+            }
```

Comment:
`else` is redundant. 

```suggestion

            return TitlePrefixRegex.Replace(title, "$2, $1$3");
```